### PR TITLE
feat: add search as root api gateway resource

### PR DESCRIPTION
**Pull request by karlknowit**
## What was solved
Adding "search" as a new Api Gateway resource path which should be used when using the booking-api service.

## How was it solved
Created a new serverless stack called search-gateway-resource which creates the new api gateway resource to the root api gateway.

## Why was it solved in this way
This is the agreed way of creating Api Gateway resources for the root Api Gateway.

## How was it tested
Tested by first deploying the root Api Gateway (gateway-root) serverless stack and thereafter deploying the new resource. Checked the Api Gateway console and verified that the new resource were attached to the root gateway.

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-1eqv1cn Lägg till search som en ny resource i root Api Gateway](https://app.clickup.com/t/1eqv1cn)

File: services/gateway/resources/search/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      Properties:
+        RestApiId: ${self:provider.apiGateway.restApiId}
+        ParentId: ${self:provider.apiGateway.restApiRootResourceId}
+        PathPart: search
+
+  Outputs:
+    ApiGatewayResourceSearch:
+      Value: !Ref ApiGatewayResourceSearch
+      Export:
+        Name: ${self:custom.stage}-ExtApiGatewayResourceSearch
```
Comment (dannil76) [inline services/gateway/resources/search/serverless.yml:30]: Missing new line :)

**Review by dannil76 — CHANGES_REQUESTED** [review]

File: services/gateway/resources/search/serverless.yml (hunk 1 above)
Comment (karlknowit) [reply services/gateway/resources/search/serverless.yml]: Added

**Review by sepehrnoor — APPROVED** [review]
Looks fine to me.

**Pull request merged**